File: app/src/main/java/com/droidtitan/volleyexamples/rest/model/air/AirQualityResponse.java
package com.droidtitan.volleyexamples.rest.model.air;

import com.google.gson.annotations.SerializedName;

public class AirQualityResponse {

    @SerializedName("consulta")
    AirQualityHolder airQualityHolder;

    public AirQuality getAirQuality() {
        return airQualityHolder.getAirQuality();
    }

    public int getHourOfReport() {
        return airQualityHolder.getHourOfReport();
    }

    public Weather getWeather() {
        return airQualityHolder.getWeather();
    }

    public Uv getUv() {
        return airQualityHolder.getUv();
    }

    public Zones getZones() {
        return airQualityHolder.getZones();
    }
}

File: app/src/main/java/com/droidtitan/volley/model/air/AirQualityResponse.kt
package com.droidtitan.volley.model.air

import com.google.gson.annotations.SerializedName

class AirQualityResponse {

  @SerializedName("consulta")
  var airQualityHolder: AirQualityHolder? = null

  fun category(): String? {
    return airQualityHolder?.airQuality?.category
  }

  fun getTemperature(): String? {
    return airQualityHolder?.weather?.temperature
  }
}
